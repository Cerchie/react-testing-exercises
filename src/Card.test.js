import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it('should render', () => {
    render(<Card src="https://static1.squarespace.com/static/55f009c8e4b079158f302bab/55f00a13e4b031210ec8b811/5649c0cae4b02d8d6e27efae/1455290053315/?format=1500w" caption="test caption"/>)
  });

  it('should match snapshot', () => {
    const {asFragment} =render(<Card src="https://static1.squarespace.com/static/55f009c8e4b079158f302bab/55f00a13e4b031210ec8b811/5649c0cae4b02d8d6e27efae/1455290053315/?format=1500w" caption="test caption"/>)
    expect(asFragment()).toMatchSnapshot()
});