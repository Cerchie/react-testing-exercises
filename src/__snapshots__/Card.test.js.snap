// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test caption
    </h4>
    <img
      alt="test caption"
      class="Card-image"
      src="https://static1.squarespace.com/static/55f009c8e4b079158f302bab/55f00a13e4b031210ec8b811/5649c0cae4b02d8d6e27efae/1455290053315/?format=1500w"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
